hooks/pre-command

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 set -euo pipefail
 
+PLUGIN_NAME=sauce-connect-buildkite-plugin
+
 # Allow to pass TMP_DIR for testing purposes
 if [[ -z "${TMP_DIR:-}" ]]; then
-  TMP_DIR=$(mktemp -d -t sauce-connect-buildkite-plugin.XXXXXXXXXX)
+  TMP_DIR=$(mktemp -d -t "${PLUGIN_NAME}.XXXXXXXXXX")
   export BUILDKITE_PLUGIN_SAUCE_CONNECT_TMP_DIR="${TMP_DIR}"
   function cleanup {
```

Comment:
Once we have a fixed directory, we won't be creating new directories constantly so we don't need to clean it up either. In fact cleaning it up will defeat the caching logic.

Reply:
3f86ef34997f59fb047fa3a20d309d1134bd87cf

Reply:
6a061661808e45b8649bdb42d9036820e54209e9